src/modules/EventModule.cpp

```diff
@@ -0,0 +1,792 @@
+			configRD.m_TicketIds.push_back(config.currentTicketIndex);
+		}
+
+		authorCurrency.setCurrency(authorCurrency.currency() - totalFee);
+		QString ticketEmoji = QString(utility::consts::emojis::TICKETS);
+		QString ticketAmount = QString::number(configRD.m_TicketIds.size());
+		client.createMessage(message.channelId(), ticketEmoji + " **You own** `" + ticketAmount + "` **ticket(s) now!**\n *(Buy them more to increase your chance)*");
+	});
+	RegisterCommand(Commands::EVENT_TICKET, "ticket", [this](Client& client, const Message& message, const Channel& channel)
+	{
+		QStringList args = message.content().split(' ');
+		if (args.size() != 1)
+		{
+			client.createMessage(message.channelId(), "**Wrong Usage of Command!** ");
+			return;
+		}
+		auto& configRD = raffleDrawGuildList[channel.guildId()][raffleDrawGetUserIndex(channel.guildId(), message.author().id())];
+		auto& config = getServerEventData(channel.guildId());
+		if (configRD.m_TicketIds.size() == 0)
+		{
+			client.createMessage(message.channelId(), "**You have no tickets.**\nYou can buy them by using `!buyticket <amount>`");
+			return;
+		}
+		QString name = UmikoBot::Instance().GetName(channel.guildId(), message.author().id());
+		QString totalTicket = QString::number(configRD.m_TicketIds.size());
+		QString ticketEmoji = QString(utility::consts::emojis::TICKETS);
+		QString ticketAmount = QString::number(configRD.m_TicketIds.size());
+		QString txt = QString(ticketEmoji + "Total Ticket(s) belonging to **%1**: `%2`\nYour ticket number(s) are:\n").arg(name, totalTicket);
+
+		for (auto ticket : configRD.m_TicketIds)
+		{
+			if (ticket == configRD.m_TicketIds.last())
+			{
+				txt += QString("%1").arg(ticket);
+				client.createMessage(message.channelId(), txt);
+				return;
+			}
+			txt += QString("%1, ").arg(ticket);
+		}
+	});
+	RegisterCommand(Commands::EVENT_SET_TICKET_PRICE, "setticketprice", [this](Client& client, const Message& message, const Channel& channel)
+	{
+		QStringList args = message.content().split(' ');
+
+		UmikoBot::VerifyAndRunAdminCmd(client, message, channel, 2, args, true, [this, &client, channel, message, args]()
+			{
+				auto& config = getServerEventData(channel.guildId());
+				config.raffleDrawTicketPrice = args.at(1).toInt();
+				client.createMessage(message.channelId(), "Raffle Draw ticket price set to **" + QString::number(config.raffleDrawTicketPrice) + "**");
+			});
+	});
+	RegisterCommand(Commands::EVENT_SET_USER_MAX_TICKET, "setusermaxticket", [this](Client& client, const Message& message, const Channel& channel)
+	{
+		QStringList args = message.content().split(' ');
+
+		UmikoBot::VerifyAndRunAdminCmd(client, message, channel, 2, args, true, [this, &client, channel, message, args]()
+			{
+				auto& config = getServerEventData(channel.guildId());
+				config.maxTicketOfUser = args.at(1).toInt();
+				client.createMessage(message.channelId(), "User max ticket set to **" + QString::number(config.maxTicketOfUser) + "**");
```

Comment:
There aren't any global and user level max ticket amounts, so the "User" here isn't necessary.